packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -58,6 +61,32 @@ type Props = {|
  * The "standard" modal layout: a titlebar, a content area, and a footer.
  */
 export default class StandardModal extends React.Component<Props> {
+    renderCloseButton() {
```

Comment:
Why is this duplicated from ModalPanel? Shouldn't we just handle this rendering in one place? And if we're doing it here why aren't we also doing it in OneColumnModal and TwoColumnModal?

Reply:
Whoops, that's a pretty big oversight on my part.  You are correct, we need to do this in the other modals as well.  Thanks for catching this.

Reply:
I double checked an we aren't using Toolbar for either `OneCloumnModal` or `TwoColumnModal` so this should be fine as is.

Reply:
We should also be using Toolbar for the footer, but I'll do that as a separate PR.